### ISSUE TITLE
Possible to show the name of the module that took the longest to build?

### ISSUE THREAD

**Issue by weijian19391** [stephencookdev/speed-measure-webpack-plugin#87]
When using the plugin with options.outputFormat === humanVerbose, it shows me the range like the following:
```
css-loader, and
sass-loader took 57,290 ms
  median       = 27,161 ms
  mean         = 30,044 ms
  s.d.         = 13,699.813 ms
  range        = (3,487 ms --> 52,241 ms)
  module count = 92
```

I assume that the range shows the module that build the fastest and the slowest, how do i go about tracking the slowest module? Can smp show me the module name?

**Comment by stephencookdev** [comment]
Yeah, you assume correctly about what that range is showing you. So you won't be able to get that information out from `humanVerbose` right now, you could get it out of the `json` blob, but it'll be a lot harder to make heads or tails of.

It wouldn't be too hard at all to make SMP log that information out, if you look at the code that logs out the range, there. If you have time to look at a PR for that, I would appreciate that 😄 Here's where you'd be able to get the data easily, and push it through https://github.com/stephencookdev/speed-measure-webpack-plugin/blob/730f3c017c47b2e994c4d622ab684e87c9653b3c/utils.js#L80
